articles/Introduction-to-time-series/index.md

```diff
@@ -0,0 +1,60 @@
+
+#### Time-series Analysis
+Time series analysis can be defined as getting meaningful insight and characteristics of the data to comprehend it better. Time series is beneficial in making predictions. It understands the scope of nature and uses it for future forecasting.
+
+#### Time Series Forecasting methods 
+Now let’s look at four time-series forecasting methods and have a brief description of them.
+1. Autoregressive Integrated Moving Average (ARIMA): This method combines both the Autoregression and Moving Average model. ARIMA models the next step in the sequence as a linear function based on the difference in observation and residual errors at previous steps. This method works best for univariate time series with trend and without seasonal components.
+2. Seasonal Autoregressive Integrated Moving-Average(SARIMA): the SARIMA models the next steps in a linear function style of the different observations, errors, differenced seasonal observation, and seasonal errors at the previous time steps. The SARIMA is fitted for univariate time series with the trend or seasonal components.
+3. Vector Autoregression (VAR): This method models the next step in each of the time series using the AR methods. It is the generalization of AR to multiple parallel time series e,g multivariate time series.
+4. Simple Exponential smoothing(SES): In this method, the model uses the next step as an exponentially weighted linear function of observation at previous time steps. This method works best for univariate time series without trend and seasonal components.
+
+#### Types of Time series
+We are going to look at two types of time series:
+
+**Deterministic time series:** A deterministic time series is expressed by an analytic expression. It doesn’t have random or probabilistic aspects. In deterministic time series, the past and future are explicitly specified by these derivatives' values at that time.
+
+**Non-deterministic time series:** This type of time-series means it cannot be described by an analytic expression and has some random aspect, making it unable to describe the behavior. They are two reasons that make a time series non-deterministic. First is if all the information required to describe it is not available secondly if the nature of the process for generation is inherent.
+
+#### Components of Time Series
+Here are some useful time series analysis techniques that can help you understand a dataset, and they are divided into four parts.
+
+- Level: This is the value for the series if it was a straight line
```

Comment:
An image representing these components pictorially would enhance the understanding. Currently, the value for series if it was a straight line sentence does not make sense without the visual aid.

Reply:
It would be better if you can include an image that has all these 4 components in it. Else, you may include 4 different images.